refinery/ui/source/js/dashboard/partials/collaboration-card.html

```diff
@@ -32,43 +32,46 @@ <h3>
              aria-expanded="false" aria-controls="collapseOne">
              <span ng-if="group.name==='Public'">
                &nbsp;<i class="fa fa-globe"></i>
+                &nbsp;{{ group.name }}
+             </span>
+             <span ng-if="group.name!=='Public'">
+                &nbsp;{{ group.name }}
              </span>
-             &nbsp;{{ group.name }}
            </a>
             <div class="col-md-4 col-xs-4">
                <span ng-if="group.can_edit">Manager</span>
+               <span ng-if="!group.can_edit"> &nbsp;</span>
             </div>
             <div class="col-md-2 col-xs-3">
-              <span ng-show="group.name !== 'Public'">
-                 <a
-                ng-click="$ctrl.openGroupEditor(group)"
-                class="no-underline pointer p-r-1-2">
-                <i class="fa fa-cog"></i>
-              </a>
-              <a ng-if="group.can_edit"
-                ng-click="$ctrl.openGroupMemberAdd(group)"
-                class="no-underline pointer">
-                 <span
-                   class="fa-stack user-add-icon"
-                   refinery-tooltip
-                   refinery-tooltip-container="body"
-                   refinery-tooltip-placement="right"
-                   title="Invite user to group">
-                   <i class="fa fa-plus-circle fa-stack-1x fa-stack-right-corner"></i>
-                   <i class="fa fa-user-o"></i>&nbsp;
-                 </span>
+              <span ng-if="group.name !== 'Public'">
+                <a
+                  ng-click="$ctrl.openGroupEditor(group)"
+                  class="no-underline pointer p-r-1-2">
+                  <i class="fa fa-cog"></i>
+                </a>
+                <a ng-if="group.can_edit" ng-click="$ctrl.openGroupMemberAdd(group)" class="no-underline pointer">
+                   <span
+                     class="fa-stack user-add-icon"
+                     refinery-tooltip
+                     refinery-tooltip-container="body"
+                     refinery-tooltip-placement="right"
+                     title="Invite user to group">
+                     <i class="fa fa-plus-circle fa-stack-1x fa-stack-right-corner"></i>
+                     <i class="fa fa-user-o"></i>&nbsp;
+                   </span>
+                </a>
               </span>
-              </a>
+              <span ng-if="group.name==='Public'"> &nbsp;</span>
```

Comment:
Issue with empty divs/grid https://stackoverflow.com/questions/35283053/empty-columns-in-responsive-grid